tests/specs/smoke.ts

```diff
@@ -218,6 +218,14 @@ const files = {
 
 	'file.txt': 'hello',
 
+	'import-typescript-parent.js': sourcemap.tag`
+	import './import-typescript-child.js';
+	`,
+
+	'import-typescript-child.ts': sourcemap.tag`
+	console.log('imported');
```

Comment:
Make this `allowJs` or something more unique that can be matched from within the `'from .js'` test